testsuite/model/README.md

```diff
@@ -48,3 +48,34 @@ mvn test -Pjpa -Dtest=ClientModelTest \
 ```
 
 The results are available in the `target/profile.html` file.
+
+Usage of Testcontainers
+-----------------------
+
+Some profiles within model tests require running 3rd party software, for example, database or Infinispan. For running these we are using [Testconainers](https://www.testcontainers.org/). This may require some additional configuration of your container engine.
```

Comment:
```suggestion
Some profiles within model tests require running 3rd party software, for example, database or Infinispan. For running these we are using [Testcontainers](https://www.testcontainers.org/). This may require some additional configuration of your container engine.
```